src/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ include(${UMF_CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
 # TODO: Cleanup the compile definitions across all the CMake files
 set(UMF_COMMON_COMPILE_DEFINITIONS UMF_VERSION=${UMF_VERSION})
 
-if(UMF_BUILD_LEVEL_ZERO_PROVIDER)
+if(UMF_BUILD_LEVEL_ZERO_PROVIDER AND UMF_FETCH_LEVEL_ZERO)
```

Comment:
I think we should add the "else" path that checks if cmake include path contains L0 headers. 
Something like:

`
if (UMF_BUILD_LEVEL_ZERO_PROVIDER) 
  if (UMF_FETCH_LEVEL_ZERO) 
    // fetch
  else
    // check and print errors
`